nomad/structs/structs.go

```diff
@@ -3963,7 +3963,7 @@ func (event *TaskEvent) PopulateEventDisplayMessage() {
 	case TaskLeaderDead:
 		desc = "Leader Task in Group dead"
 	default:
-		desc = ""
+		desc = event.Message
```

Comment:
Will event or message ever be nil? If the message is empty, should there be some generic output?

Reply:
Early return in line 3855 handles event being nil. If the message is empty the existing behavior leaves it as is, that's fine for now since the generic output can't add any more new information to the user.